### PR TITLE
First full version of fair_tracks JSON schema with example file

### PR DESCRIPTION
**Pull request by sveinugu**

### CODE REVIEW

File: json/schema/fair_genomic_tracks.schema.json [hunk 1]
```diff
@@ -0,0 +1,420 @@
+{
+    "$schema": "http://json-schema.org/draft-06/hyper-schema#",
```
Comment (dtitov) [inline json/schema/fair_genomic_tracks.schema.json]: I've faced a problem with this line when tested this schema in TrackFind. I'm using [this library](https://github.com/everit-org/json-schema) to deal with JSON schema and its documentation says:
> - if there is "$schema": "http://json-schema.org/draft-04/schema" in the schema root, then Draft 4 will be used
> - if there is "$schema": "http://json-schema.org/draft-06/schema" in the schema root, then Draft 6 will be used
> - if there is "$schema": "http://json-schema.org/draft-07/schema" in the schema root, then Draft 7 will be used
> - if none of these is found then Draft 4 will be assumed as default

So if something custom is placed there (like `hyper-schema`) this library throws an exception. I'm not sure what's the expected behavior and if this is correct, but we need to deal with it somehow.

Comment (sveinugu) [reply json/schema/fair_genomic_tracks.schema.json]: As far as I can understand, we do not need hyper-schema. JSON hyper-schema seems to be a way to design more advanced APIs where for instance the returned JSON contains links to how the content can be used in the API. A bit like the move from structs to objects, if I understand correctly. Seems very useful, but perhaps not that much used yet? Here is an example that is relatively easy to understand (most other explanations were very hard to read for a JSON nwebie): http://benfalk.com/blog/2016/02/09/json-hyper-schema/

I see no reason why we should have a hyper-schema, and suggest we change into a regular schema. It is probably a mistake.
```suggestion
    "$schema": "http://json-schema.org/draft-06/schema#",
```

**Comment by sveinugu** [comment]
@jmfernandez Have you looked at this yet?

**Comment by jmfernandez** [comment]
Hi @sveinugu , yes, I have been having a look at the generated JSON Schema and content. I'm using it just to fine tune the OWL terms validation, and I found that the EFO IRI is not working (http://www.ebi.ac.uk/efo/releases/2018-02-15/efo.owl should be https://www.ebi.ac.uk/efo/releases/v2018-02-15/efo.owl).

I'm adding checks for this, in order to validate whether the ontology is reachable

**Comment by jmfernandez** [comment]
I have just also realized that the generated JSON Schema has as `$id` the value `https://www.elixir-europe.org/IS/FAIRGenomicTracks/json-schemas/0.1` , meanwhile the basic_example.json has as `fair_tracks._schema` the value `https://www.elixir-europe.org/IS/FAIRGDataTracks/json-schemas/0.1`

**Comment by jmfernandez** [comment]
Hi again. I have just finished the development of the extensions for Python in main branch, and I have also documented them: https://github.com/inab/GenomicDataTracks_JSON_schemas/tree/master/toolsForValidation

So, you can start validating. I have also added the JSON Schema partially fixed and a JSON to be validated, also partially fixed.

**Comment by sveinugu** [comment]
Replaced by https://github.com/inab/GenomicDataTracks_JSON_schemas/pull/2

**Pull request closed without merging**